proxmin/utils.py

```diff
@@ -6,6 +6,8 @@
 def get_spectral_norm(L):
     if L is None:
         return 1
+    elif hasattr(L, "spectral_norm"):
+        return L.spectral_norm()
```

Comment:
@fred3m Can you make this a property call instead of a function call.

Reply:
Fixed, and also replaced `flatten` with `reshape`, since `flatten` makes a copy while `array.reshape(-1)` is a view to a flat version of `array`.